tests/unit/test_api.py

```diff
@@ -305,7 +305,7 @@ def test_merger_handles_list_deletions():
     ]
 
     merged, conflict = merge(root, head, update, head_source='arxiv')
-    assert merged == expected_merged
+    assert sorted(merged.items()) == sorted(expected_merged.items())
```

Comment:
isn't this basically a less efficient way to do the same thing? two dicts are equal iff they have the same keys and the two values for a given key are equal.

Reply:
Yes, you are right